modules/rtm/tests/invert_prospect.R

```diff
@@ -0,0 +1,77 @@
+library(devtools)
+#install('~/Projects/pecan/pecan/modules/rtm')
```

Comment:
`devtools::install` is convenient because it reinstalls the package and attempts to reload it within the same R session, which is nice for developing because it lets me run tests and devlop code in the same session. `devtools::load_all` is an even more development-specific function that loads everything inside of a package, regardless of whether or not it is exported by the NAMESPACE. It's sort of like `source` ing every script in the R folder, plus their external package dependencies. I'm pretty sure you should never use `devtools::load_all` outside of testing.